test_shopinvader_payment/tests/test_quotation.py

```diff
@@ -22,40 +21,47 @@ def test_get_no_cart_payment_info(self):
 class ShopinvaderQuotationPaymentCase(CommonConnectedQuotationCase):
```

Comment:
here you test the shopinvader_payment_quotation addon. Therefore, the file name should be `test_shopinvader_payment_qutation.py` and the classname `TestShopinvaderPaymentQuotation`